scripts/m0

```diff
@@ -88,11 +88,11 @@ _make() {
 
     if [ "${TARGET:-}" = rpms ]; then
         ## Tolerate warnings. rpms get built in spite of them.
-        make ${MAKE_OPTS:-} $TARGET || exit $?
+        bear make ${MAKE_OPTS:-} $TARGET || exit $?
```

Comment:
why "bear?" Is "bear" installed by default?

Reply:
sorry, this change was pushed accidentally. I will remove it. Thanks!